pkg/network/link/names.go

```diff
@@ -21,10 +21,17 @@ package link
 
 import (
 	"fmt"
+	"strconv"
+	"strings"
 )
 
 func GenerateTapDeviceName(podInterfaceName string) string {
-	return "tap" + podInterfaceName[3:]
+	trimmedName := strings.TrimPrefix(podInterfaceName, "net")
```

Comment:
This mess makes me think we should have left the `net` as a prefix (or a different one), just to be consistent with other interfaces that relate to the same network.

Reply:
I agree, we have a chance to reconsider and align the naming scheme:
external pod interface name:  `pod<hash>` (added by multus).
bridge: `br<hash>
tap device: `tap<hash>`
dummy: `k8s<hash>` / `nic<hahs>`

@AlonaKaplan @EdDev WDYT?

Reply:
- I would prefer all to have a 3 chars prefix.
- For this PR, I would just update the pod interface with this `pod` prefix, the others we can handle in a followup.

Reply:
Since we're changing (I still don't understand exactly what problem we're trying to solve ...) I agree having always a prefix with the same length for all network infra in the pod is good (helps when grepping).